x/crisis/spec/02_messages.md

```diff
@@ -11,12 +11,7 @@ corresponding updates to the state.
 
 Blockchain invariants can be checked using the `MsgVerifyInvariant` message. 
 
-```go
-type MsgVerifyInvariant struct {
-	Sender         sdk.AccAddress 
-	InvariantRoute string
-}
-```
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0-rc7/proto/cosmos/crisis/v1beta1/tx.proto#L14-L22
```

Comment:
Wouldn't it be nice for the messages doc to have the generated msg types, also is there a reason to prefer links over using proto messages directly

Reply:
FYI such links should just render as code samples on https://docs.cosmos.network/

Reply:
I agree it's slightly harder to read the raw md file, but I also believe it's less error-prone and easier to maintain.

Reply:
yeah I see where you're coming from, was just concerned about the readability when using github directly.